### ISSUE TITLE
Vagrant 1.7.0 Debian package fails to package boxes

### ISSUE THREAD

**Issue by exarkun** [hashicorp/vagrant#4944]
After upgrading to 1.7.0:

```
Running vagrant package --output my-package.box
/opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/machine.rb:153:in `action': wrong number of arguments (2 for 1) (ArgumentError)
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/plugins/commands/package/command.rb:83:in `package_vm'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/plugins/commands/package/command.rb:72:in `block in package_target'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/plugin/v2/command.rb:226:in `block in with_target_vms'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/plugin/v2/command.rb:220:in `each'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/plugin/v2/command.rb:220:in `with_target_vms'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/plugins/commands/package/command.rb:70:in `package_target'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/plugins/commands/package/command.rb:44:in `execute'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/cli.rb:42:in `execute'
        from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/environment.rb:301:in `cli'
        from /opt/vagrant/bin/../embedded/gems/gems/vagrant-1.7.0/bin/vagrant:174:in `<main>'
```

The same operation completes successfully on Vagrant 1.6.5.

**Comment by mastermindg** [comment]
Save issue here on 1.7 - Ubuntu 14.04 host, Virtualbox 4.3.20

**Comment by sethvargo** [comment]
This is fixed in 1.7.2

**Comment by mastermindg** [comment]
Perfect, thx!
